test/functional/apps/visualize/_inspector.js

```diff
@@ -58,7 +58,7 @@ export default function ({ getService, getPageObjects }) {
           await PageObjects.visualize.selectAggregation('Terms');
           await PageObjects.visualize.selectField('machine.os.raw');
           await PageObjects.visualize.setSize(2);
-          await PageObjects.visualize.toggleOtherBucket();
+          await PageObjects.visualize.toggleOtherBucket(3);
```

Comment:
I'm confused why you need to provide an index here. Isn't there only one "other" bucket toggle being shown?